### PR TITLE
Limit update

### PR DESCRIPTION
**Pull request by dionboles-asym**
added limit and offset and the gradle task for pyproject.toml

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,11 +1,9 @@
 server.port=8080
 
-cda.bqTable=broad-dsde-dev.cda_dev
+cda.bqTable=gdc-bq-sample.dev
```
Comment (rfricke-asymmetrik) [inline src/main/resources/application.properties:3]: Revert this change please.

File: src/main/resources/application-staging.properties [hunk 2]
```diff
@@ -1,10 +1,9 @@
 server.port=8080
 
-cda.bqTable=broad-dsde-staging.cda_staging
+cda.bqTable=gdc-bq-sample.dev
```
Comment (rfricke-asymmetrik) [inline src/main/resources/application-staging.properties:3]: Revert this change please.

File: src/main/resources/application-prod.properties [hunk 3]
```diff
@@ -1,9 +1,9 @@
-server.port=8080
+sserver.port=8080
 
-cda.bqTable=broad-dsde-prod.cda_prod
+cda.bqTable=gdc-bq-sample.dev
```
Comment (rfricke-asymmetrik) [inline src/main/resources/application-prod.properties:3]: Revert this change please.

File: src/main/resources/application-dev.properties [hunk 4]
```diff
@@ -1,11 +1,9 @@
 server.port=8080
 
-cda.bqTable=broad-dsde-dev.cda_dev
+cda.bqTable=gdc-bq-sample.dev
```
Comment (rfricke-asymmetrik) [inline src/main/resources/application-dev.properties:3]: Revert this change please.

File: src/main/resources/application-alpha.properties [hunk 5]
```diff
@@ -1,10 +1,9 @@
 server.port=8080
 
-cda.bqTable=broad-dsde-alpha.cda_alpha
+cda.bqTable=gdc-bq-sample.dev
```
Comment (rfricke-asymmetrik) [inline src/main/resources/application-alpha.properties:3]: Revert this change please.

File: src/main/java/bio/terra/cda/app/generators/SqlGenerator.java [hunk 6]
```diff
@@ -140,11 +142,28 @@ protected QueryContext buildQueryContext(
...
+    buildLabels(query.getR(), queryMap);
+  }
+
+  public QueryJobConfiguration.Builder generate()
+      throws IllegalArgumentException, JsonProcessingException {
     String querySql = sql(qualifiedTable, rootQuery, false, false, false);
     QueryJobConfiguration.Builder queryJobConfigBuilder =
         QueryJobConfiguration.newBuilder(querySql);
-
+    Map<String, String> qMap = new HashMap<>();
```
Comment (rfricke-asymmetrik) [inline src/main/java/bio/terra/cda/app/generators/SqlGenerator.java:164]: These are no longer necessary because we are not using the labels anymore.

**Review by rfricke-asymmetrik — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 7]
```diff
@@ -255,4 +263,36 @@ sonarqube {
   }
 }
 
+abstract class BUILD_PYPROJECT_TOML extends DefaultTask{
+	@TaskAction
+	def greet(){
```
[the remark this answers is not in the document]
Comment (dionboles-asym) [reply build.gradle:268]: I just updated it

File: src/main/resources/application.properties (hunk 1 above)
Comment (dionboles-asym) [reply src/main/resources/application.properties]: updated

File: src/main/resources/application-staging.properties (hunk 2 above)
Comment (dionboles-asym) [reply src/main/resources/application-staging.properties]: updated

File: src/main/resources/application-prod.properties (hunk 3 above)
Comment (dionboles-asym) [reply src/main/resources/application-prod.properties]: updated

File: src/main/resources/application-dev.properties (hunk 4 above)
Comment (dionboles-asym) [reply src/main/resources/application-dev.properties]: updated

File: src/main/resources/application-alpha.properties (hunk 5 above)
Comment (dionboles-asym) [reply src/main/resources/application-alpha.properties]: updated

File: src/main/java/bio/terra/cda/app/generators/SqlGenerator.java (hunk 6 above)
Comment (dionboles-asym) [reply src/main/java/bio/terra/cda/app/generators/SqlGenerator.java]: updated and fixed

**Review by rfricke-asymmetrik — APPROVED** [review]

**Review by ahaessly — APPROVED** [review]
👍
